charts/keyrock/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: keyrock
-version: 0.6.2
+version: 0.7.0
```

Comment:
can be left out, will automatically be increased, depending on the label

Reply:
done